<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据集详情</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>基本信息</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>标签位置</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>图片位置</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="browseLabelPath">
        <property name="text">
         <string>浏览…</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>名称</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="browseImagePath">
        <property name="text">
         <string>浏览…</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="imagePathEdit"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="labelPathEdit"/>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="showImagePathButton">
        <property name="toolTip">
         <string>如果没有反应，说明您填写的路径不存在</string>
        </property>
        <property name="text">
         <string>在系统中显示</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QLineEdit" name="nameEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="update_info">
        <property name="toolTip">
         <string>只有点击该按钮后，您在上方的更改才会生效</string>
        </property>
        <property name="text">
         <string>更新信息</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="showLabelPathButton">
        <property name="toolTip">
         <string>如果没有反应，说明您填写的路径不存在</string>
        </property>
        <property name="text">
         <string>在系统中显示</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="dataTypeLabel">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="toolTip">
      <string>无法点击的按钮就是还没做的（）</string>
     </property>
     <property name="title">
      <string>操作</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_5">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>标注…</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="copyDatasetButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>复制…</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButton_7">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>合并…</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="divideButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>划分为训练与验证…</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton_9">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>转换为…</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="deleteDatasetButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>删除…</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>提取部分分类…</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="cleanDatasetButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>清理…</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="exportButton">
        <property name="text">
         <string>压缩并导出…</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameEdit</tabstop>
  <tabstop>imagePathEdit</tabstop>
  <tabstop>browseImagePath</tabstop>
  <tabstop>showImagePathButton</tabstop>
  <tabstop>labelPathEdit</tabstop>
  <tabstop>browseLabelPath</tabstop>
  <tabstop>showLabelPathButton</tabstop>
  <tabstop>update_info</tabstop>
  <tabstop>pushButton_5</tabstop>
  <tabstop>copyDatasetButton</tabstop>
  <tabstop>pushButton_7</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>divideButton</tabstop>
  <tabstop>pushButton_9</tabstop>
  <tabstop>deleteDatasetButton</tabstop>
  <tabstop>cleanDatasetButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
